src/flow.ts

```diff
@@ -220,67 +216,13 @@ export class Flow {
    *
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
-  static setMusicFont(...fontNames: string[]): Font[] {
-    // #FIXME: HACK to facilitate the VexFlow 5 migration.
-    // HACK-BEGIN
-    // Introduce the correct font stacks step by step.
-    switch (fontNames[0]) {
-      case 'Bravura':
-        CommonMetrics.fontFamily = 'Bravura,Roboto Slab';
-        break;
-      case 'Gonville':
-        CommonMetrics.fontFamily = 'GonvilleSmufl,Bravura,Roboto Slab';
-        break;
-      case 'Leland':
-        CommonMetrics.fontFamily = 'Leland,Bravura,Roboto Slab';
-        break;
-      case 'Petaluma':
-        CommonMetrics.fontFamily = 'Petaluma,Bravura,Petaluma Script';
-        break;
-      case 'MuseJazz':
-        CommonMetrics.fontFamily = 'MuseJazz,Bravura,Academico';
-        break;
-      case 'Gootville':
-        CommonMetrics.fontFamily = 'Gootville,Bravura,Academico';
-        break;
-      case 'Finale Ash':
-        CommonMetrics.fontFamily = 'Finale Ash,Bravura,Academico';
-        break;
-      case 'Finale Maestro':
-        CommonMetrics.fontFamily = 'Finale Maestro,Bravura,Academico';
-        break;
-      case 'Finale Broadway':
-        CommonMetrics.fontFamily = 'Finale Broadway,Bravura,Academico';
-        break;
-      default:
-        CommonMetrics.fontFamily = fontNames.join(',');
-    }
-    // HACK-END
-
+  static setMusicFont(...fontNames: string[]): void {
     // Convert the array of font names into an array of Font objects.
-    const fonts = fontNames.map((fontName) => Font.load(fontName));
-    Tables.MUSIC_FONT_STACK = fonts;
-    Glyph.MUSIC_FONT_STACK = fonts.slice();
-    Glyph.CURRENT_CACHE_KEY = fontNames.join(',');
-    return fonts;
-  }
-
-  /**
-   * Used with vexflow-core which supports dynamic font loading.
-   */
-  // eslint-disable-next-line
-  static async fetchMusicFont(fontName: string, fontModuleOrPath?: string | FontModule): Promise<void> {
-    // The default implementation does nothing.
-    // See vexflow-core.ts for the implementation that vexflow-core.js uses.
+    CommonMetrics.fontFamily = fontNames.join(',');
```

Comment:
But this handles more than just the music engraving font, correct? 

In a separate PR we could consider renaming this method. Vex.Flow.setMusicFont() is not appropriate if it is to set the entire font stack. It should just be called `setFont()`.

I still think it would be worth considering having two separate font stacks. One for text and one for music symbols. Right now if you inspect the SVG output, you see lots of Roboto Slab declarations where they are not even appropriate.

So we could have a `setMusicFont()` and a `setTextFont()` that set the appropriate font families.

Then on Elements that have mixed music glyphs and text characters, we could join the two together into one long font stack.

For now, your approach is fine... but I think it's good to keep this idea as an option, if we run into any issues mixing music and text font names.

Reply:
I believe, that should keep a single stack, otherwise we have to decide what we are rendering. It is better that the browser selects. The multiple definition of the fonts in the SVG is a cleanup problem, already resolved in #100. There is still one not resolved cleanup problem with the font size.